objects/src/transaction/consumed_notes.rs

```diff
@@ -91,9 +91,16 @@ impl ToAdviceInputs for ConsumedNotes {
     }
 }
 
+impl From<ConsumedNotes> for Vec<ConsumedNoteInfo> {
+    fn from(consumed_notes: ConsumedNotes) -> Self {
+        consumed_notes.notes.into_iter().map(|note| note.into()).collect::<Vec<_>>()
+    }
+}
+
 // CONSUMED NOTE INFO
 // ================================================================================================
 
+// TODO: Should we rename this to NoteTombstone?
```

Comment:
I'm not against it - but sound a bit grim :)

Reply:
true :) I think the struct becomes redundant once we transition to `pcall` so it's probably not worth it.